### DISCUSSION TITLE
Ignore ordered lists

### DISCUSSION THREAD

**Discussion by luantrasel** [remarkjs/remark#994] [Q&A] [answered]
Hi there,

Is there a way to ignore ordered lists?

I tried some options (incrementListMarker, bulletOrdered) on remark-stringify but they didn't fix my problem. I'd like to completely ignore ordered lists. There's a way to do that?

This is how I currently parse my texts (with markdowns) into html

```
import remark from 'remark'
import html from 'remark-html'
import emoji from 'remark-emoji'
import breaks from 'remark-breaks'
import externalLinks from 'remark-external-links'

export default (content, options) => {
    const openLinksInNewWindow = options?.openLinksInNewWindow || false

    const result = remark()
        .use(emoji, { emoticon: true })
        .use(breaks)
        .use(externalLinks, {
            target: openLinksInNewWindow ? '_blank' : false,
            rel: [openLinksInNewWindow ? 'noopener noreferrer' : ''],
        })
        .use(html)
        .processSync(content)

    return result.contents
}
```

input example:

```
1. Item one
2. Item two
```

**current** output:
```
<ol>
   <li>Item one</li>
   <li>Item two</li>
</ol>
```

**expected** output:

```
<p>1. Item one</p>
<p>2. Item two</p>
```

**Comment by ChristianMurphy** [comment, marked as answer]
What do you mean by "ignore lists"? (Do you want them removed entirely? to be left as raw plain text? something else?)
Could you share an example for a given input, what you'd want the output to be?

**Reply by luantrasel** [reply]
Yes, I would like them to be left as raw plain text. And since I'm parsing it to html using remark-html the final output would render the items as regular text inside `<p>` instead of `<li>`

I updated my question with input, current output and expected output

**Reply by ChristianMurphy** [reply]
See https://github.com/remarkjs/remark/discussions/757 for how to disable things, including potentially lists.
In particular the linked issue https://github.com/zestedesavoir/zmarkdown/issues/416#issuecomment-982812961

And (potentially) https://github.com/micromark/micromark/discussions/31

**Reply by luantrasel** [reply]
Thanks for your answer. I managed to ignore lists using according to this [comment](https://github.com/zestedesavoir/zmarkdown/issues/416#issuecomment-982812961) as you suggested.

Then the problem changed and I realized we had only to ignore ordered lists, which doing as the comment says didn't work.

So I ended up changing the text before parsing it to something that wouldn't become a list and at the end I replaced it back to regular content
